stdlib/strings/strings.flux

```diff
@@ -1,9 +1,7 @@
+// The strings package provides functions to manipulate UTF-8 encoded strings.
 package strings
 
 
-// The Flux strings package provides functions to manipulate UTF-8 encoded strings.
-//
-//
```

Comment:
I just noticed that each of the comments is off by one. Meaning the comment for title is actually commenting toUpper. Can you fix that?